trio_asyncio/_loop.py

```diff
@@ -344,7 +342,7 @@ async def _waitpid(self, pid, callback, *args):
 
     def add_child_handler(self, pid, callback, *args):
         """Add a callback to run when a child process terminates."""
-        h = self._loop.run_trio(self._waitpid, pid, callback, *args)
+        h = self._loop.trio_as_future(self._waitpid, pid, callback, *args)
```

Comment:
This line was never tested before. And I have no idea how to test it now. But the change is equivalent to what it was before. Here's how (removed) `_loop.run_trio` was defined:

```python
    @deprecated("0.10.0", issue=38, instead="trio_as_aio(proc)(*args)")
    def run_trio(self, proc, *args):
        return self.trio_as_future(proc, *args)
```